web/modules/custom/portland/modules/portland_ecouncil/portland_ecouncil.module

```diff
@@ -157,3 +164,50 @@ function portland_ecouncil_field_group_form_process_build_alter(array &$element,
     _hide_deprecated_field_values($form);
   }
 }
+
+const AGENDA_ITEM_DISPOSITION_ASSIGNED_TO_COMMITTEE_TID = 984;
+
+/**
+ * Implements hook_ENTITY_TYPE_update().
+ *
+ * When an agenda item is updated, update the associated council document's Assigned committee to match, if:
+ * - item disposition is "Assigned to Committee"; and
+ * - council document is not null; and
+ * - committee is not null; and
+ * - committee and is different from previous value
+ */
+function portland_ecouncil_relation_update(EntityInterface $relation) {
+  $council_doc = $relation->field_council_document->entity;
+  $original_committee = $relation->original->field_committee->entity;
+  $committee = $relation->field_committee->entity;
+  if ($relation->field_agenda_item_disposition->target_id === (string) AGENDA_ITEM_DISPOSITION_ASSIGNED_TO_COMMITTEE_TID
```

Comment:
Would it help to define the const as a string so we don't need to cast here? Very minor improvement. :)

Reply:
For context, the target_id in the entity reference field storage is a string for some reason, while in other use cases e.g. the $form entity reference options, it's an integer.

Since earlier in the file we use a const to unset a form option which is an integer, I thought it would be good to be consistent and make all the consts numbers, even if it's a bit uglier to cast it. What do you think?